libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -84,7 +84,7 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
     // @Values: 0:None,1:AUTO,2:uBlox,3:MTK,4:MTK19,5:NMEA,6:SiRF,7:HIL,8:SwiftNav,9:UAVCAN,10:SBF,11:GSOF,13:ERB,14:MAV,15:NOVA,16:HemisphereNMEA,17:uBlox-MovingBaseline-Base,18:uBlox-MovingBaseline-Rover,19:MSP,20:AllyStar,21:ExternalAHRS
     // @RebootRequired: True
     // @User: Advanced
-    AP_GROUPINFO("_TYPE",    0, AP_GPS, _type[0], HAL_GPS_TYPE_DEFAULT),
+    AP_GROUPINFO_FLAGS("_TYPE",    0, AP_GPS, _type[0], HAL_GPS_TYPE_DEFAULT, AP_PARAM_FLAG_ENABLE),
```

Comment:
Let's remove the enable on the GPS_TYPE param because it is very commonly used.

Reply:
removed GPS changes